pkgs/tools/inputmethods/ibus-engines/ibus-typing-booster/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, python3, ibus, pkgconfig, gtk3, makeWrapper, m17n_lib
+, pango, gdk_pixbuf, atk
+}:
+
+let
+
+  python = python3.withPackages (ps: with ps; [
+    pygobject3
+    dbus-python
+  ]);
+
+  mkRepoPaths = stdenv.lib.makeSearchPath "lib/girepository-1.0";
+
+in
+
+stdenv.mkDerivation rec {
+  name = "ibus-typing-booster-${version}";
+  version = "2.1.1";
+
+  src = fetchFromGitHub {
+    owner = "mike-fabian";
+    repo = "ibus-typing-booster";
+    rev = version;
+    sha256 = "01kpxplk9nh56f32fkq3nnsqykbzpi7pcxbfp38dq0prgrhw9a6b";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkgconfig makeWrapper ];
+  buildInputs = [ python ibus gtk3 m17n_lib ];
+
+  postInstall = ''
+    for i in $out/bin/emoji-picker $out/libexec/ibus-setup-typing-booster $out/libexec/ibus-engine-typing-booster; do
+      wrapProgram "$i" \
```

Comment:
Why not use `wrapGAppsHook`?

Reply:
It definitely needs to be used, otherwise the booster crashes with “Settings schema 'org.freedesktop.ibs.engine.typing-booster' is not installed”